src/RegistrationController.js

```diff
@@ -58,19 +62,65 @@ function (
     save: Okta.loc('registration.form.submit', 'login')
   });
 
+  var DEFAULT_CALLBACK_ERROR = 'We could not process your registration at this time. Please try again later';
+
   return BaseLoginController.extend({
     className: 'registration',
     initialize: function() {
       var clientId = this.options.settings.get('clientId');
       var registrationApi = this.options.settings.get('baseUrl')+'/api/v1/registration/'+clientId;
 
-      var Schema = RegistrationSchema.extend({
-        url: registrationApi+'/form'
-      });
-      
-      var schema = new Schema();
+      var doPostSubmit = _.bind(function () {
```

Comment:
In general, when using private helper functions, pull them out into the outer scope and pass in the context as a variable, i.e.

```javascript
function doPostSubmit(controller) {
  if (controller.model.get('activationToken')) {
    // etc
  }
}
```